src/validator/view-schemas/v2.js

```diff
@@ -113,6 +113,11 @@ const definitions = {
         '$ref': '#/definitions/arrayOptions'
       },
 
+      // Text to display at bottom of cell
+      cellMessage: {
```

Comment:
Probably needs a better name more descriptive name. Is this only used for sections?

Reply:
Yeah only for sections, would something like `sectionText`, `sectionMessage`, or `sectionFootnote` work?

Reply:
Going with `sectionFootnote`